db/migrate/20191211172832_add_status_to_profiles.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToProfiles < ActiveRecord::Migration[5.2]
+  def change
+    add_column :profiles, :status, :string
```

Comment:
can do default: "pending" instead of assigning it on step 1
`,default: "pending"`